tasks/set_root_hints.yml

```diff
@@ -7,7 +7,7 @@
   with_items:
     - "{{ total_nodes.stdout_lines }}"
 
-- name: set root hints based on model (740XD G-CL)
+- name: set root hints based on model
```

Comment:
Are we setting root hints for all models here? If that's the case, maybe `name: set root hints` is a better name?